float32 vx
float32 vy
float32 vz
float32 yaw
float32 yaw_rate
string frame_id
bool update_frame
bool auto_arm
---
bool success
string message
